src/Common/ExecCommand.php

```diff
@@ -77,6 +98,33 @@ public function printed($arg)
     }
 
     /**
+     * Should command output be printed
+     *
+     * @param bool $arg
+     *
+     * @return $this
+     */
+    public function printOutput($arg)
+    {
+        return $this->printed($arg);
```

Comment:
Could you reverse this logic, and have `printed` call `printOutput`? @DavertMik would like`printed()` to emit a warning via the logger if someone calls it, but this clearly should not happen if someone calls the recommended method (`printOutput`).